tests/jest/views/ProjectsTab.spec.js

```diff
@@ -281,9 +286,30 @@ describe('ProjectsTab.vue Test', () => {
 			expect(workPackages).toMatchSnapshot()
 		})
 	})
+
+	describe('action menu', () => {
+		it.only('should display the action menu in linked work packages', async () => {
+			wrapper = mountWrapper(false)
+			await wrapper.setData({
+				workpackages: workPackagesSearchResponse,
+			})
+			await localVue.nextTick()
+			let toggleMenu = wrapper.find(toggleMenuSelector)
+			await toggleMenu.find('button').trigger('click')
```

Comment:
```suggestion
			expect(wrapper.find(menuPopOverSelector).isVisible()).toBeFalsy()
			await toggleMenu.find('button').trigger('click')
```
It would be more solid if you check for the action buttons existence before clicking the toggler.